github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -389,15 +392,15 @@ postsubmits:
               - |
                 cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
                 cd images
-                ./publish_image.sh vm-image-builder quay.io kubevirtci
+                ./publish_multiarch_image.sh vm-image-builder quay.io kubevirtci
             # docker-in-docker needs privileged mode
             securityContext:
               privileged: true
             resources:
               requests:
-                memory: "1Gi"
+                memory: "29Gi"
```

Comment:
I am not sure this will be able to run against the control-plane cluster - the memory request is too high for these nodes so the job will never be scheduled. I think this has to be updated to run against the `kubevirt-prow-workloads` cluster

https://github.com/kubevirt/project-infra/pull/2631/files#diff-681acd437a2a7b11c454c006e2fab1606fc0b436c79674066b686608c609e434R384

The presubmit works on the `ibm-prow-jobs` cluster but that is just because it is only building the x86_64 image.

Reply:
What should I modify? Reduce the memory? or change the cluster from `ibm-prow-jobs` to `kubevirt-prow-workloads`?

Reply:
I would recommend changing this to run on the large workloads cluster `kubevirt-prow-workloads`